scripts/schemas.js

```diff
@@ -36,17 +36,23 @@ const main = async () => {
     console.log(`--> Running command for ${target}\n`)
 
     try {
-      const { stdout, stderr } = await promisify(exec)(
+      await promisify(exec)(
         `nx run-many --target=${target} --all --with-deps --parallel --maxParallel=6 ${
           skipCache ? '--skip-nx-cache' : ''
         }`,
       )
-
-      console.log(stdout)
-
-      if (stderr) {
-        console.error(`Error while running generate-schemas: ${stderr}`)
-      }
+        .then((res) => {
+          console.log('-res', res)
+        })
+        .catch((e) => {
+          console.log('-e', e)
+        })
+
+      // console.log(stdout)
```

Comment:
Please remove this 🙏

Reply:
Not my code. Jeremy is debugging broken builds.